sled-agent/types/src/rack_ops.rs

```diff
@@ -5,16 +5,16 @@
 use omicron_uuid_kinds::{RackInitUuid, RackResetUuid};
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
+use strum::{EnumCount, EnumIter, IntoEnumIterator};
 
 /// Current status of any rack-level operation being performed by this bootstrap
 /// agent.
-#[derive(
-    Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema,
-)]
+#[derive(Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
 #[serde(tag = "status", rename_all = "snake_case")]
 pub enum RackOperationStatus {
     Initializing {
         id: RackInitUuid,
+        step: RssStep,
```

Comment:
Idle thought with Dave's [what API interfaces exist](https://github.com/oxidecomputer/meta/blob/master/engineering/reconfigurator/upgrade/interfaces.adoc) question in my head: I wonder if this should be more opaque than an explicit enum. If wicket is only going to display the step, I could see an argument to making this `String` (or maybe `Vec<String>` if we wanted to get fancy with multiline descriptions).

My thought there is that if we make changes to RSS that change / reorder / add / delete `RssStep`s, both wicketd and sled-agent would have to be updated to the same version for them to know about those changes. But now that I say that, I guess we always run RSS with a consistent version on all sleds, so that's already true anyway?

Reply:
I had considered just sending a string, but then I would also want to send wicket a current step and max step too, as the x/y is useful as well. 

And, yeah, the wicketd and sled-agent should be the same for RSS, at least on the same sled.  If other sleds have different versions, then RSS itself would have to deal with that.